cypress/support/utils/commonUtils.ts

```diff
@@ -0,0 +1,15 @@
+// cypress/support/utils.js
+
+// Generate a random string of a specific length
+export function generateRandomString(length: number) {
+    let result = '';
+    const characters = 'ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789';
+  
+    for (let i = 0; i < length; i++) {
+      const randomIndex = Math.floor(Math.random() * characters.length);
+      result += characters.charAt(randomIndex);
+    }
+  
+    return result;
```

Comment:
```suggestion
    return Array.from({ length : length }, () => Math.random().toString(36)[2]).join('');
```

Так короче. И можно поместить в `commands.ts`